spotguide/spotguide.go

```diff
@@ -151,46 +126,93 @@ func (r LaunchRequest) RepoFullname() string {
 	return r.RepoOrganization + "/" + r.RepoName
 }
 
-func downloadGithubFile(githubClient *github.Client, owner, repo, file, tag string) ([]byte, error) {
-	reader, err := githubClient.Repositories.DownloadContents(ctx, owner, repo, file, &github.RepositoryContentGetOptions{
-		Ref: tag,
-	})
+type ReleaseBodyYaml struct {
+	Pipeline string `json:"pipeline" yaml:"pipeline"`
+}
+
+type Spotguide struct {
```

Comment:
please add a comment about what does this struct represent, it's named quite ambiguously

Reply:
up